octoprint_mrbeam/iobeam/laserhead_handler.py

```diff
@@ -152,31 +153,36 @@ def _calculate_power_correction_factor(self):
 		p_65 = None
 		p_75 = None
 		p_85 = None
+		target_power = self.LASER_POWER_GOAL_DEFAULT
 		if power_calibration:
 			p_65 = power_calibration.get('power_65', None)
 			p_75 = power_calibration.get('power_75', None)
 			p_85 = power_calibration.get('power_85', None)
+			target_power = power_calibration.get('target_power', self.LASER_POWER_GOAL_DEFAULT)
+
+			if target_power < 0 or target_power >= 1300:
```

Comment:
pls define the limit in a constant. 
If target_power > 130 pls set target_power = 1300

Reply:
Changed